exercises/practice/resistor-color/.meta/example.jl

```diff
@@ -0,0 +1,20 @@
+COLORS = [
+    "black",
+    "brown",
+    "red",
+    "orange",
+    "yellow",
+    "green",
+    "blue",
+    "violet",
+    "grey",
+    "white"
+]
```

Comment:
```suggestion
const COLORS = (
    "black",
    "brown",
    "red",
    "orange",
    "yellow",
    "green",
    "blue",
    "violet",
    "grey",
    "white"
)
```

Would probably be better. Not that it matters for the example.jl, but if we include it in an approaches intro or if a student goes looking then it would be nice to avoid issues around mutable globals.